docs/api/MQTT.md

```diff
@@ -17,7 +17,7 @@ client.on('connect', function () {
 client.on('message', function (topic, message) {
   // message is Buffer
   console.log(message.toString());
-  client.end();
+  client.disconnect();
```

Comment:
The MQTT API keeps consistent with [MQTT.js](https://github.com/mqttjs/MQTT.js), which doesn't use `disconnect()`.